_build/maintainer_guide.rst

```diff
@@ -0,0 +1,332 @@
+Symfony Docs Maintainer Guide
+=============================
+
+The `symfony/symfony-docs`_ repository stores the Symfony project documentation
+and is managed by the `Symfony Docs team`_. This article explains in detail some
+of those management tasks, so it's only useful for maintainers and not regular
```

Comment:
only useful for maintainers **and contributors**?

Reply:
I'm afraid not :)  This guide only make sense to people who actually merge in the Symfony Docs repo. For the regular contributors we have this other guide: https://symfony.com/doc/current/contributing/documentation/overview.html

Reply:
Yep, noticed afterwards when there was a long section on the gh tool :+1: